migtests/scripts/functions.sh

```diff
@@ -92,7 +98,7 @@ import_schema() {
 }
 
 import_data() {
-	yb_migrate import data --export-dir ${EXPORT_DIR} \
+	yes | yb_migrate import data --export-dir ${EXPORT_DIR} \
```

Comment:
where do we require yes in import data also?

Reply:
For MySQL, when we try to import in the `public` schema, yb_migrate prompts for confirmation.

Reply:
ok, i thought it is only for import schema.